.github/workflows/docs.yml

```diff
@@ -2,10 +2,10 @@ name: build docs
 
 on:
   workflow_dispatch:
-  pull_request_target:
-  push:
-    branches:
-      - master
+  # pull_request_target:
+  # push:
+  #   branches:
+  #     - master
```

Comment:
Commenting `pull_request_target` makes sense, but I recommend the one on push master